sccm/apps/TOC.yml

```diff
@@ -63,6 +63,10 @@
       href: deploy-use/manage-apps-from-the-windows-store-for-business.md
     - name: App-V virtual environments
       href: deploy-use/create-app-v-virtual-environments.md
+    - name: Copy applications
```

Comment:
As I mentioned below, I suggest removing the "copy app" article.

Reply:
👍